pyclang/utils.py

```diff
@@ -55,9 +55,10 @@ def run_cmd(
     returncode = p.wait()
     raw_stderr = to_str(p.stderr.read())
     if returncode not in expect_returncode:
-        sys.stderr.write(raw_stderr)
-        sys.stderr.flush()
-        raise RuntimeError(f'command "{cmd_str}" failed with exitcode {returncode}')
+        sys.stderr.write(f'Error: Command "{cmd_str}" failed with exit code {returncode}\n')
+        if raw_stderr:
+            sys.stderr.write(f'Details:\n{raw_stderr}\n')
+        sys.exit(returncode)
```

Comment:
No flush is needed in this case?

Reply:
With `sys.exit()`, an explicit flush should not be needed because Python automatically flushes sys.stderr before exiting.

However, since I switched back to raising an exception (SystemExit instead of RuntimeError), I reintroduced sys.stderr.flush(). This ensures that error messages are immediately visible, especially in case SystemExit is caught in the future.